docs/lib/content/configuring-npm/package-json.md

```diff
@@ -818,11 +818,12 @@ to express this. If you depend on features introduced in 1.5.2, use
 
 ### peerDependenciesMeta
 
-When a user installs your package, npm will emit warnings if packages
-specified in `peerDependencies` are not already installed. The
-`peerDependenciesMeta` field serves to provide npm more information on how
+The `peerDependenciesMeta` field serves to provide npm more information on how
 your peer dependencies are to be used. Specifically, it allows peer
-dependencies to be marked as optional.
+dependencies to be marked as optional. Npm would not automatically install
+optional peer dependencies. This allows you to
```

Comment:
> Npm would not automatically install optional peer dependencies.

This is a very confusing and potentially misleading sentence.  At a glance it reads that npm does not install optional peer dependencies. In fact it does, it just does not raise an error if they can not be installed.

Reply:
Well then my memory of the situation. I must have gotten optional and optional peer dependencies confused.